ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ProfileSelection.tsx

```diff
@@ -142,11 +190,13 @@ function ProfileSelection({ profiles, isFetchingProfiles }: ProfileSelectionProp
                                     isSelected: allRowsSelected,
                                 }}
                             />
-                            <Th>Name</Th>
-                            <Th>Description</Th>
+                            <Th />
```

Comment:
Are we presenting the list of profiles to users in the same way we receive it from the backend? I was wondering if it would make it easier for users if it was sorted alphabetically.

Reply:
Definitely a good suggestion. I was planning on doing sorting/filtering/pagination in a separate PR once I wrap up the remaining blockers.